packages/paste-core/components/skeleton-loader/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {render} from '@testing-library/react';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {SkeletonLoader} from '../src';
+import {Default} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+describe('SkeletonLoader', () => {
+  it('should render', () => {
+    const {getByTestId} = render(<Default />);
+    expect(getByTestId('default-skeleton')).toBeDefined();
+    expect(getByTestId('default-skeleton')).toHaveStyleRule('background-color', 'colorBackgroundStrong');
+    expect(getByTestId('default-skeleton')).toHaveStyleRule('border-radius', 'borderRadius20');
+    expect(getByTestId('default-skeleton')).toHaveStyleRule('height', 'sizeIcon20');
+  });
+
+  it('should render layout prop styles', (): void => {
```

Comment:
I don't think I've seen us use `(): void` in our tests before. What is it used for here?

Reply:
This is just setting the return type to void, which basically means the function doesn't return any data.

It may not be necessary since the `it` function from Jest comes with the void return type anyway, but its good practice to set the function return type. I think we've done it pretty sporadically throughout the monorepo.